libs/fgdbg/client/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+import org.jetbrains.compose.compose
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.6.10"
+    id("org.jetbrains.compose") version "1.1.1"
+}
+
+group = "com.google.filament"
+version = "1.0"
+val ktorVersion: String by project
+
+repositories {
+    google()
+    mavenCentral()
+    maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+  maven {
```

Comment:
Indentation is incorrect on this maven block

Reply:
good catch!